core/controller/src/main/scala/whisk/core/controller/actions/SequenceActions.scala

```diff
@@ -353,6 +369,24 @@ protected[actions] trait SequenceActions {
 
     /** Max atomic action count allowed for sequences */
     private lazy val actionSequenceLimit = whiskConfig.actionSequenceLimit.toInt
+
+    /** Starts the tracing for the sequence action and creates the tracing metadata hash map into which
+     *  the tracer implementation  inserts its own ids, these ids are then propagated via the ActivationMessage into
+     *  invoker that can add the reference from the primitive action to the parent sequence action
+     */
+    private def startTracingHelper(cause: Option[ActivationId],
```

Comment:
we should consider if this is fully going to subsume the markers generated in the logs if we want to have two way of tracing or just one because the information is going to be duplicated.